presto-main/src/main/java/com/facebook/presto/operator/aggregation/AggregationUtils.java

```diff
@@ -136,12 +137,28 @@ public static Type getOutputType(@Nullable Method outputFunction, AccumulatorSta
         }
     }
 
+    public static TypeSignature getOutputTypeSignature(@Nullable Method outputFunction, AccumulatorStateSerializer<?> serializer)
+    {
+        if (outputFunction == null) {
```

Comment:
I think we should better check it above, and don't accept any nulls here. If you think that it is still better to have it here, than at least pass `outputFunction` as an optional to explicitly declare that this method know what to do with the cases where output function is not defined.